libsodium-sys/build.rs

```diff
@@ -146,12 +146,12 @@ fn build_libsodium() {
     println!("cargo:rustc-link-lib=static=libsodium");
     if is_release_profile() {
         println!(
-            "cargo:rustc-link-search=native={}/msvc/Win32/Release/v140/",
+            "cargo:rustc-link-search=native={}/msvc/Win32/Release/v142/",
             get_crate_dir()
         );
     } else {
         println!(
-            "cargo:rustc-link-search=native={}/msvc/Win32/Debug/v140/",
+            "cargo:rustc-link-search=native={}/msvc/Win32/Debug/v142/",
```

Comment:
Why 142?

Reply:
Isn't it a some kind of a version of a library? I just took the latest one which is 142.

Reply:
https://habr.com/ru/company/microsoft/blog/439664/

Reply:
So that means that older msvc compiler won't be able to use this version? An we still want to use 140?

Reply:
> We want to keep the momentum going and make sure that you have a similarly successful adoption experience with MSVC v142 too. This is why we’re announcing today that our team is committed to provide binary compatibility for MSVC v142 with both MSVC v141 and v140.
This means that if you compile all your code with the v142 toolset but still have one or more libraries that are built with the v140 or v141 toolset, linking all of it together (with the latest linker) will work as expected. To make this possible, VC Runtime does not change its major version in VS 2019 and remains backward compatible with previous VC Runtime versions.

There is a backward compatibility, but I don't see anything about forward compatibility.

Reply:
ok, reverted to 140.